### ISSUE TITLE
best strategy for benchmark / testing assets

### ISSUE THREAD

**Issue by samhatchett** [OpenWaterAnalytics/EPANET#159]
Two options I can think of:

- Amazon S3 for benchmark inp-binary artifacts
- separate Github Repo for [example networks](https://github.com/OpenWaterAnalytics/epanet-example-networks) with results files as "release" artifacts

thoughts?

**Comment by michaeltryby** [comment]
This is a great suggestion! I like the idea of a separate repo slightly better than S3 only because networks could be shared publicly and used for purposes than just regression testing.

**Comment by samhatchett** [comment]
I'm also liking that idea more.  `inp` files are great for mainline versioning, and then the snapshotted files and result binaries lend themselves well to release artifacts. one could even imagine a CI workflow that produces benchmark binaries for new models that get merged...

**Comment by michaeltryby** [comment]
Let's do it!  Separate repo it is!

**Comment by michaeltryby** [comment]
Doing some more thinking about this ... I figured it would be good to include the epanet.exe with the tests and the benchmarks so that everything needed to recreate the benchmark would be included and available in case it was ever needed.

To that end, I did some digging to figure out what version of Visual Studio was used to compile epanet 2.0.12 from way back in 2008.  
```
\>dumpbin /HEADERS .\epanet2012.exe
...
OPTIONAL HEADER VALUES
             10B magic # (PE32)
            6.00 linker version
           20000 size of code
...
```
Wikipedia has a [table ](https://en.wikipedia.org/wiki/Microsoft_Visual_Studio#History) with the history of linker versions. Low and behold, epanet version 2.0.12 was compiled with Visual Studio 6.0 circa 1998. Twenty years old and still running!
